fixtures/ext-types/lib/tests/test_generated_bindings.rs

```diff
@@ -27,3 +27,21 @@ fn test_exttypes_python() -> uniffi::deps::anyhow::Result<()> {
 
     uniffi::testing::run_foreign_language_testcase(&pkg_dir, &udl_files, &test_file_path)
 }
+
+#[test]
```

Comment:
ouch - I forgot about this re 4a5e7716deba31ac80823e8c3fc9d9a8dbf8ff58 - we should be very close to allowing something like `uniffi_macros::build_foreign_language_testcases!` here (either new macro for the array of udls, or maybe use arrays everywhere?) Need not block on that though